GitCommands/Git/GitModule.cs

```diff
@@ -1370,7 +1370,7 @@ public void Reset(ResetMode mode, string? file = null)
             _gitExecutable.RunCommand(GitCommandHelpers.ResetCmd(mode, commit: null, file));
         }
 
-        public string ResetFiles(IReadOnlyList<string> files)
+        public string CheckoutIndexFiles(IReadOnlyList<string> files)
```

Comment:
I see your intent here, but IMO the old name communicated the method's purpose better. Maybe the new name should mention "Force" too? E.g., `ForceCheckoutIndexFiles`....
Xml-doc would be helpful too.

Reply:
Added xml-doc, but I would like to keep the name, the behavior is different from git-reset.